docs/en/install-upgrade/redirects.asciidoc

```diff
@@ -25,4 +25,40 @@ See the {observability-guide}/whats-new.html[Observability highlights] page for
 === Highlights
 
 This page no longer exists.
-Go to {estc-welcome}/new.html[Check out the latest from Elastic].
\ No newline at end of file
+Go to {estc-welcome}/new.html[Check out the latest from Elastic].
+
+[role="exclude",id="apm-breaking-changes"]
+=== APM breaking changes
+
+This page no longer exists.
+See {apm-guide-ref}/apm-breaking.html[APM {version} breaking changes] instead.
+
+[role="exclude",id="beats-breaking-changes"]
+=== Beats breaking changes
+
+This page no longer exists.
+See {beats-ref}/breaking-changes-8.0.html[{beats} 8.0 breaking changes] instead.
+
+[role="exclude",id="elasticsearch-breaking-changes"]
+=== {es} breaking changes
+
+This page no longer exists.
+See {ref}/breaking-changes.html[{es} {version} breaking changes] instead.
+
+[role="exclude",id="security-breaking-changes"]
+=== {elastic-sec} breaking changes
+
+This page no longer exists.
+See {security-guide}/release-notes.html[{elastic-sec} release notes] instead.
+
+[role="exclude",id="kibana-breaking-changes"]
+=== {kib} breaking changes
+
+This page no longer exists.
+See {kibana-ref}/release-notes.html[{kib} release notes] instead.
+
+[role="exclude",id="logstash-breaking-changes"]
+=== {ls} breaking changes
+
+This page no longer exists.
+See {logstash-ref}/breaking-8.0.html[{ls} 8.0 breaking changes] instead.
```

Comment:
```suggestion
See {logstash-ref}/breaking-changes.html[{ls} breaking changes] instead.
```

Reply:
What about linking to the Logstash Breaking changes page?  The additional context helps users understand that we don't have breaking changes for every minor.  If we did have breaking changes for an 8.x release, a user could easily find them. 

This page would be evergreen and not require updating if/when we move to 9.x.

Reply:
Great suggestion! Done in [`277d0d8` (#2495)](https://github.com/elastic/stack-docs/pull/2495/commits/277d0d8262636db03bc28c133e78133768a4e7a9)